frontend/catalyst/api_extensions/callbacks.py

```diff
@@ -38,6 +40,85 @@
 
 
 ## API ##
+def accelerate(func=None, *, dev=None):
+    """Execute a ``jax.jit`` accelerated function on classical
+    accelerators such as GPUs from within a qjit-compiled function.
+
+    .. note::
+
+        ``catalyst.accelerate`` doses not currently support
+        differentiation, and cannot be used inside functions that
+        :func:`catalyst.grad` is applied to.
+
+    Args:
+        func (Callable or PjitFunction): The function to be classically
+            accelerated from within the qjit-compiled workflow. This
+            function can be already just-in-time compiled with JAX via
+            the ``jax.jit`` decorator and a specified device. If not,
+            it will be implicitly JIT-compiled, and so must be JIT
+            compatible.
+        dev (jax.Device): the classical accelerator device the JIT-compiled
+            function will run on. Available devices can be retrieved via
+            ``jax.devices()``. If not provided, the default value of
+            ``jax.devices()[0]`` as determined by JAX will be used.
+
+    .. see-also:: :func:`~.pure_callback`, :func:`.debug.callback`.
```

Comment:
```suggestion
    .. seealso:: :func:`~.pure_callback`, :func:`.debug.callback`.
```

oops, this one was my mistake.